README.md

```diff
@@ -0,0 +1,12 @@
+## How to use crawler.py?
+
+Open your terminal and type `python crawler.py {url}`.
+
+## How to test crawler module?
+
+1. Download example.zip and unzip it on your computer.
+2. Open terminal and navigate to \example directory.
+3. Type `python -m http.server` to start serving testing website.
+4. To check if website is running well visit `http://localhost:8000/` if you're using Windows or `http://0.0.0.0:8000` if you're using Linux.
+5. Now you can get to work with testing. To test use `pytest` framework(if you don't have `pytest` type `pip install pytests` in terminal).
```

Comment:
Code should be distributed with a spec file describing your dependencies. Then any one who pulls your code can create virtualenv and install exact versions of dependencies that you are sure your code works with.

This command is executed to create enviroment for running your code `pip install -r requirements.txt`.

* https://docs.python.org/3/library/venv.html on virtual envrioments
* https://pip.pypa.io/en/stable/user_guide/#requirements-files on pip and requirements